linchpin/InventoryFilters/InventoryProviders.py

```diff
@@ -16,14 +16,14 @@
 from .JSONInventoryFormatter import JSONInventoryFormatter
 
 filter_classes = {
-    "aws": AWSInventory,
-    "beaker": BeakerInventory,
-    "duffy": DuffyInventory,
-    "dummy": DummyInventory,
-    "gcloud": GCloudInventory,
-    "libvirt": LibvirtInventory,
-    "openstack": OpenstackInventory,
-    "ovirt": OvirtInventory,
+    "aws_ec2_res": AWSInventory,
```

Comment:
@14rcole Good job ! 
Its time we filter resources with their types in inventory 
But am not sure if we get into floating ip inventory but it should be noted